pkg/kindsys/report.json

```diff
@@ -807,6 +807,32 @@
       "pluralName": "JaegerDataSourceCfgs",
       "schemaInterface": "DataSourceCfg"
     },
+    "librarypanel": {
+      "category": "core",
+      "codeowners": [
+        "grafana/grafana-as-code",
+        "grafana/grafana-bi-squad",
+        "grafana/plugins-platform-frontend",
+        "grafana/user-essentials"
```

Comment:
Not sure based on what this is generated, but seems wrong. cc @sdboyer

Reply:
sorry i missed this earlier - @joanlopez i have noticed a couple of these ownerships that seem questionable

Reply:
It currently looks for the CODEOWNERS for each of the code generated paths (MD, Go, TS) as well as the schema (CUE) file, but that could easily be changed. What would be more appropriate here? Do we have any proper "rule" to look for owners? 🤔